primary/Tenkuu Danzai Skelter Heaven (primary).ass

```diff
@@ -161,110 +159,94 @@ Dialogue: 0,0:10:53.97,0:10:54.83,Default,,0,0,0,,And Midori!
 Dialogue: 0,0:10:54.83,0:10:56.40,Default,,0,0,0,,Follow your orders!
 Dialogue: 0,0:10:57.60,0:10:59.84,Default,,0,0,0,,Analyzing.
 Dialogue: 0,0:10:59.84,0:11:01.57,Default,,0,0,0,,Core detected in center of object.
-Dialogue: 0,0:11:01.57,0:11:05.12,Default,,0,0,0,,The entity's exterior is heavily armored,\Nso enter from underneath.{上部の表皮は硬いので、脚部まわりから侵入してください}{m - lit. "the outer skin layer/epidermis is hard up top, so enter from the leg area"}
+Dialogue: 0,0:11:01.57,0:11:05.12,Default,,0,0,0,,The entity's carapace is heavily armored,\Nso enter from underneath.{上部の表皮は硬いので、脚部まわりから侵入してください}{m - lit. "the outer skin layer/epidermis is hard up top, so enter from the leg area"}
 Dialogue: 0,0:11:08.78,0:11:09.73,Default,,0,0,0,,Are you listening, Rin?
-Dialogue: 0,0:11:09.73,0:11:11.35,Default,,0,0,0,,There's no time for emotions.
+Dialogue: 0,0:11:09.73,0:11:11.35,Default,,0,0,0,,Don't let your emotions get the best of you.
 Dialogue: 0,0:11:19.72,0:11:21.63,Default,,0,0,0,,All units will provide cover while 02 attacks.
 Dialogue: 0,0:11:32.17,0:11:33.37,Default,,0,0,0,,Rin!
-Dialogue: 0,0:11:38.38,0:11:40.69,Default,,0,0,0,,This is an artificial human.
-Dialogue: 0,0:11:40.95,0:11:46.48,Default,,0,0,0,,As it grows, its intelligence will surpass a human's,
-Dialogue: 0,0:11:46.48,0:11:48.38,Default,,0,0,0,,along with its physical abilities.{m - lit. "and the same goes for its physical abilities"}
+Dialogue: 0,0:11:38.38,0:11:40.69,Default,,0,0,0,,This is a homunculus.
```

Comment:
"Homunculus" has a different meaning in the scientific and technical space than in the pseudoscience space. I vote to revert.
```suggestion
Dialogue: 0,0:11:38.38,0:11:40.69,Default,,0,0,0,,This is an artificial human.
```

Reply:
"Synthetic", perhaps? Might convey a bit more meaning.